src/app/app-reducers/effects/cases.app.effects.spec.ts

```diff
@@ -1,9 +1,8 @@
-import { CoreModule } from '@ansyn/core';
 import { EffectsRunner, EffectsTestingModule } from '@ngrx/effects/testing';
 import { async, inject, TestBed } from '@angular/core/testing';
 import { CasesAppEffects } from './cases.app.effects';
 import { SelectOverlayAction, UnSelectOverlayAction } from '@ansyn/overlays';
-import { CasesService } from '@ansyn/menu-items/cases';
+import { CasesService, casesConfig, CasesConfig } from '@ansyn/menu-items/cases';
```

Comment:
CasesConfig is not in use in this file.
Please remove the import from CaseConfig.
Change to `import { CasesService, casesConfig } from '@ansyn/menu-items/cases';`